src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,30 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        AuthenticationService authenticationService = new AuthenticationServiceImpl(userService);
+
+        User newRegisteredUser = null;
+        try {
+            System.out.println("=========================================");
+            newRegisteredUser = authenticationService.register("mary@mail.com", "111111");
+        } catch (Exception e) {
+            throw new RegistrationException("",e);
+        }
+        System.out.println(newRegisteredUser);
+
+        try {
+            System.out.println("=========================================");
+            System.out.println(authenticationService.login("bob@mail.com", "123"));
+        } catch (Exception e) {
+            throw new AuthenticationException("",e);
```

Comment:
Similar to the previous comment, catching a generic `Exception` is not recommended. Consider catching specific exceptions that the `login` method might throw to improve error handling.

Reply:
ok, done